wp3/Exercise_4_keyboardScanning/keyboardScanning_without_keypadLibrary/keyboardScanning_without_keypadLibrary.ino

```diff
@@ -0,0 +1,153 @@
+// (C) Emma Litvin, Nicole Quinstedt. Group: 11 (2023)
+// Work package 3
+// Exercise 3
+// Submission code: XXXX
+
+/*--------------------------------------------------------------------------------------------
+Exercise 3: The program reads the keyboard input and prints out the key number in the serial 
+  monitor if a key is pressed. If no key is pressed, nothing is printed out.
+
+  Two solutions are presented:
+  1. using pinMode(),
+  2. using digitalWrite().
+---------------------------------------------------------------------------------------------*/
+
+/*--------------------------------------------------------------------------------------------
+  Sources:
+  - Alternative solution using bits:
```

Comment:
I dont think we need to have this link, we didnt use. Was more like a fun fact for alternative solution.